src/app/configuration/configuration-service/configuration.service.spec.ts

```diff
@@ -61,4 +62,11 @@ describe('ConfigurationService', () => {
             );
         }
     });
+
+    it('should be able to set camera view', () => {
+        const service: ConfigurationService = TestBed.get(ConfigurationService);
+        expect(service.getSceneUsePerspectiveCamera()).toBeTruthy();
+        service.setSceneUsePerspectiveCamera(false);
+        expect(service.getSceneUsePerspectiveCamera()).toBeFalsy();
+    });
```

Comment:
I would extend the test case, as it only checks whether it can be toggled off, not on.

```suggestion
        service.setSceneUsePerspectiveCamera(true);
        expect(service.getSceneUsePerspectiveCamera()).toBeTruthy();
    });
```

Reply:
Addressed